etg/pyevent.py

```diff
@@ -138,6 +150,18 @@ def run():
             return clone
             """)
 
+    cls.addPyMethod("__setstate__", "(self, state)",
+        doc = """Sets internal state of PyEvent object to allow pickling""",
+        body = """\
+            self.__dict__.update(state)
+            """)
+
+    cls.addPyMethod("__getstate__", "(self)",
+        doc = "Returns internal state. Precense enables pickling",
```

Comment:
Oops. 

Also, I'm still not 100% confident with etg, or whats going on cpp side / py side. 

The get/set state methods allow pickle, well, pickle the object. Because otherwise it can't be sure of the full-state of objects that overwrite __getattr__

Should the state control methods utilize the cpp side state using _getAtrDict()?